content/docs/command-reference/move.md

```diff
@@ -103,8 +145,8 @@ $ tree
 
 We use `dvc add` to track a file with DVC, then we use `dvc move` to change its
 location. If target path already exists and is a directory, data file is moved
```

Comment:
Pre-existing grammar issue:
```suggestion
location. If the target path already exists and is a directory, data file is moved
```
> This may break the formatting, sorry.